src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ListViewAdapter.cs

```diff
@@ -623,6 +623,10 @@ internal List<Cell> GetCellsFromPosition(int position, int take)
 		void OnCollectionChanged(object sender, NotifyCollectionChangedEventArgs e)
 		{
 			OnDataChanged();
+			if(e.Action == NotifyCollectionChangedAction.Reset)
+			{
+				_realListView.SmoothScrollToPosition(0);
```

Comment:
Add a comment to explain the purpose of scrolling to position 0 when the action is Reset.

Reply:
> Add a comment to explain the purpose of scrolling to position 0 when the action is Reset.

@jsuarezruiz  Added a comment to explain why the scroll position is set to the 0th position.